go/vt/vtgate/planbuilder/abstract/subquery.go

```diff
@@ -34,10 +34,13 @@ var _ Operator = (*SubQuery)(nil)
 
 // SubQueryInner stores the subquery information for a select statement
 type SubQueryInner struct {
-	Inner           Operator
-	Type            engine.PulloutOpcode
-	SelectStatement *sqlparser.Select
-	ArgName         string
+	Inner            Operator
+	Type             engine.PulloutOpcode
+	SelectStatement  *sqlparser.Select
+	ArgName          string
+	HasValues        string
```

Comment:
a few of these fields are not very self explanatory. we should probably add a couple of comments explaining why we store them here